homeassistant/components/denonavr/sensor.py

```diff
@@ -0,0 +1,74 @@
+"""Support for Denon AVR additional sensors."""
+
+from typing import Any
+
+from denonavr import DenonAVR
+from denonavr.const import POWER_ON
+
+from homeassistant.components.sensor import SensorEntity
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.helpers.typing import StateType
+
+from .device import DenonDeviceEntity
+
+ATTR_ZONE = "zone"
+
+SENSOR_BASS = "bass"
+SENSOR_BASS_LEVEL = "bass_level"
+SENSOR_DYNAMIC_EQ = "dynamic_eq"
+SENSOR_DYNAMIC_VOLUME = "dynamic_volume"
+SENSOR_MULTI_EQ = "multi_eq"
+SENSOR_REFERENCE_LEVEL_OFFSET = "reference_level_offset"
+SENSOR_TONE_CONTROL_ADJUST = "tone_control_adjust"
+SENSOR_TONE_CONTROL_STATUS = "tone_control_status"
+SENSOR_TREBLE = "treble"
+SENSOR_TREBLE_LEVEL = "treble_level"
+
+SENSOR_ENTITIES = {
+    SENSOR_BASS: "Bass",
+    SENSOR_BASS_LEVEL: "Bass Level",
+    SENSOR_DYNAMIC_EQ: "Dynamic EQ",
+    SENSOR_DYNAMIC_VOLUME: "Dynamic Volume",
+    SENSOR_MULTI_EQ: "Multi EQ",
+    SENSOR_REFERENCE_LEVEL_OFFSET: "Reference Level Offset",
+    SENSOR_TONE_CONTROL_ADJUST: "Tone Control Adjust",
+    SENSOR_TONE_CONTROL_STATUS: "Tone Control Status",
+    SENSOR_TREBLE: "Treble",
+    SENSOR_TREBLE_LEVEL: "Treble Level",
+}
```

Comment:
This will hard code names. We should use translation keys. Even tho every sensor would only need 2 fields (or 3 if you add the value function as well), I recommend looking at entity descriptions for this.